README.md

```diff
@@ -14,14 +22,21 @@ npm install @yourpass/api-js-client --save
 
 ## Usage
 
-This chapter describe basic usage of client. You can find more examples [here](/examples).
+This chapter describes basic usage of client. You can find more examples [here](/examples).
 
 ### Create OAuth2 client
 
-This code below describe, how create client, which is connected in to API via OAuth2.
+First, you have to create client and authenticate using OAuth2.
+
+For authentication You need:
+
+* clientId: OAuth client id. You can get it on [Yourpass Console Profile Page](https://console.yourpass.eu/user/detail).
+* clientSecret: OAuth client secret. You can get it on [Yourpass Console Profile Page](https://console.yourpass.eu/user/detail)
```

Comment:
Tohle asi není pravda, podle mě by se ten clientId/client secret měl objevit to spíše v dokumentaci (ten univerzální). https://doc.yourpass.eu/#introduction a pokud bude  každý uživatel vidět univerzální nebo seznam všech klientů...tak bychom to tam měli nejdříve dodělat....